iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -121,6 +121,9 @@ public void configureJobConf(TableDesc tableDesc, JobConf jobConf) {
     if (tableDesc != null && tableDesc.getProperties() != null &&
         tableDesc.getProperties().get(WRITE_KEY) != null) {
       jobConf.set("mapred.output.committer.class", HiveIcebergOutputCommitter.class.getName());
+      // this will turn off job committing on the Tez AM side, so that we can commit the write jobs
+      // on HS2 side instead using the HiveIcebergMetaHook
+      jobConf.set("hive.tez.mapreduce.output.committer.class", "");
```

Comment:
nit: is there a constant for the key? Also I have seen `NullOutCommitter` class around somewhere. Could we use that here?

Reply:
Good idea, will try the NullOutCommitter

Reply:
I replaced the string key with the constant. As for using a null committer, that class would need to extend the `OutputCommitter` from the `tez-api` package. Currently iceberg-handler does not depend on Tez, so we can only achieve that if we introduce that dependency.

Reply:
Then using `NullOutCommitter` does not worth it